src/modules/orders/order.repository.ts

```diff
@@ -47,14 +48,32 @@ class OrderRepository extends Repository<Order> {
   }
 
   #toCell = (cell: ReturnType<typeof parseOrderCell>) =>
-    cell.orderAmount === BigInt(0)
+    this.isInvalidOrderCell(cell)
       ? null
       : this.create({
           ...cell,
           output: JSON.stringify(cell.output),
           type: cell.type === '00' ? OrderType.Bid : OrderType.Ask,
           price: cell.price.toString(16).padStart(16, '0'),
         })
+
+  private isInvalidOrderCell(cell: ReturnType<typeof parseOrderCell>) {
+    const smallestCapacity: bigint =
+      (cell.orderAmount * cell.price * BigInt(1003)) / BigInt(1000) / BigInt(10 ** 13) + BigInt(17900000000)
+    const biggestCapacityOrderAmount: bigint =
+      ((cell.sudtAmount - (cell.sudtAmount * BigInt(3)) / BigInt(1000)) * cell.price) / BigInt(10 ** 10)
+
+    return (
+      cell.orderAmount === BigInt(0) ||
```

Comment:
`cell.orderAmount === BigInt(0)` can be changed to `!cell.orderAmount` for concision.

And don't too much chaining `||`, return `false` in each falsy branch.